content/applications/hr/recruitment/time_in_stage.rst

```diff
@@ -0,0 +1,82 @@
+======================
+Time in stage analysis
+======================
+
+The *Time In Stage Analysis* report provides information on how long applicants stay in each stage
+of the recruitment process. This is important, as every job position has specific :ref:`process
+details <recruitment/new_job_position/edit>` that state the length of time applicants should expect
+to wait between specific stages.
+
+Knowing how long applicants remain in each stage can help highlight possible bottlenecks. Analyzing
+this data allows the recruitment team to assess each stage, identify any issues, and pivot their
+strategies to move applicants through each stage, within the expected time interval.
+
+Time in stage analysis report
+=============================
+
+To access the report, navigate to :menuselection:`Recruitment app --> Reporting --> Time in Stage
+Analysis`. By default, the report displays data from all job positions, with the stages populating
+the x-axis, and the number of days populating the y-axis, in a line chart.
+
+The default filter is :guilabel:`Last 365 Days Applicant`, showing information for the last 365
+days.
+
+Hover over a stage in the line chart to reveal a popover window listing all the job positions within
+it, and the average number of days each job position sits in each stage.
+
+.. image:: time_in_stage/time-in-stage.png
+   :align: center
+   :alt: The default Time In Stage Analysis report.
+
+Bar chart view
+--------------
+
```

Comment:
Can remove this heading. I dont see a need for this to be it's own section since it is pretty much a continuation of the previous instruction. This also keeps the in-page table of contents free to jump right to the "use case" section
```suggestion
```